docs/how-to-guides/new-user-guides/backup-restore-and-disaster-recovery/backup-restore-and-disaster-recovery.md

```diff
@@ -91,10 +91,10 @@ A migration is performed by following [these steps.](migrate-rancher-to-new-clus
 
 ## Default Storage Location Configuration
 
-Configure a storage location where all backups are saved by default. You will have the option to override this with each backup, but will be limited to using an S3-compatible or Minio object store.
+Configure a storage location where all backups are saved by default. Various configuration options are available, including specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` helm chart. You will also have the option to override the configured storage location with each backup, but will be limited to using an S3-compatible or Minio object store.
```

Comment:
```suggestion
Configure a default storage location for your backups. There are various configuration options, such as specifying an S3-compatible object store as the location for individual backups or choosing an existing `StorageClass` during installation of the `backup-restore-operator` Helm chart. You also have the option to override the configured storage location with each backup, but are limited to using an S3-compatible or Minio object store.
```

Avoiding passive voice and future tense when not necessary

Reply:
Thanks for the review Marty! Pushed the changes to all versions and will make a note to maintain present tense in the future.